src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,7 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+import './UserInfo.scss';
+
+export const UserInfo = ({ user: { email, name } }) => (
```

Comment:
You should consider adding PropTypes to validate the types of values the component props should hold. This is useful for catching bugs by providing type checking.